### DISCUSSION TITLE
3D map

### DISCUSSION THREAD

**Discussion by LTKingChaos** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#573] [Ideas]
Hi

Are you going to include the new 3D map?

**Comment by PiotrMachowski** [comment]
What 3D map do you mean?

**Comment by LTKingChaos** [comment]
In the updated version of xiaomi map there is a 3d map 
![3dmapRoborock](https://user-images.githubusercontent.com/98852503/232220847-5352865d-001d-4891-91df-5094e8cdd514.jpg)

**Reply by PiotrMachowski** [reply]
Ok, this card just displays an image of a map. Map is downloaded from Xiaomi Servers by [Map Extractor](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor). Card supports both 2d and 3d images of maps, but I don't think that Map Extractor will support 3d maps

**Reply by LTKingChaos** [reply]
Ok. Thanks for quick reply
